dev-setup/README.md

```diff
@@ -0,0 +1,232 @@
+# Teracy-dev-setup
+
+Please follow this getting started guide to set up the development environment.
+
+Please take a cup of coffee with you, you mostly don't have to do anything but wait for the result, enjoy!
+
+
+## Getting Started
+
+- Set up `teracy-dev` if not yet, follow: http://dev.teracy.org/docs/develop/getting_started.html .
```

Comment:
duplicated of https://github.com/teracyhq/dev-setup/blob/develop/README.md

Reply:
fixed!